examples/minimal-react/playwright.config.ts

```diff
@@ -14,8 +15,10 @@ const config: PlaywrightTestConfig = {
     ...devices['Desktop Chrome'],
     baseURL: baseUrl,
     headless: opts.headless,
+    video: process.env['CI'] ? 'retain-on-failure' : 'off',
+    screenshot: process.env['CI'] ? 'only-on-failure' : 'off',
```

Comment:
there seems to be a race condition in the e2e testing, and enabling recording slows down playwright enough to fix it... Obviously not the right fix, but I can't reproduce it locally or with recording enabled to see what's happening 🙈